esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,15 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (lang.exists("Exceptions.Exception", response)) {
```

Comment:
These exceptions will have been published to the toaster already via ESPRequest?